p2p/pex/pex_reactor.go

```diff
@@ -414,6 +431,14 @@ func (r *PEXReactor) crawlPeersRoutine() {
 	}
 }
 
+// hasPotentialPeers indicates if there is a potential peer to connect to, by
+// consulting the Switch as well as the AddrBook.
+func (r *PEXReactor) hasPotentialPeers() bool {
+	o, i, d := r.Switch.NumPeers()
```

Comment:
I'd prefer whole words here... `outgoing, incoming, dialing = `

Reply:
Was debating that when writing it, here the lifcycle of the vars is the next line, which was the original idea for single character variables being advocated for in go. Will change, wondering if it will add much for readers to understand better, tho.

Reply:
I think it's fine the way it is since it gets consumed right away and the words are otherwise large.

That said I think we're supposed to avoid single letter names. Maybe `out, in, dial` would be nice. I guess at that point, adding an `ing` to everything is no biggie. Meh.

Reply:
Will expand them single characters, like the shorter version as well.